kinto/tests/test_views_collections.py

```diff
@@ -35,12 +35,15 @@ def test_collections_name_should_be_simple(self):
                           headers=self.headers,
                           status=400)
 
-    def test_unknown_bucket_raises_404(self):
+    def test_unknown_bucket_raises_403(self):
```

Comment:
The link you pasted doesn't buy us much: it says it's a good practice and it's a bad practice.

[The HTTP spec](http://pretty-rfc.herokuapp.com/RFC2616#status.403) states:

>  If the server does not wish to make this information available to the client, the status code 404 (Not Found) can be used instead. 

I think that's for us to make a choice here: do we want to disclose this information or not?

A 403 has one advantage over a 404: it's clear that the resource isn't accessible by the connected user, so for instance, if she's not connected with the right account she can find it out and act properly.

It also have a disadvantage:  It leaks the fact that the resource exists to another user, which could then use this information in many different ways.